src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.courses.domain;
+
+import java.util.List;
+
+public class Image {
+    private static final long LIMIT_FILE_SIZE = 1_048_576;
+
+    private final byte[] data;
+    private final ImageType fileType;
+    private final Size size;
+
+    public Image(byte[] data, Size size, String extension){
+        this(data, size, ImageType.from(extension));
+    }
+
+    public Image(byte[] data, Size size, ImageType fileType) {
+        if(data.length > LIMIT_FILE_SIZE){
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
        if (data.length > LIMIT_FILE_SIZE) {
```